src/riscv.h

```diff
@@ -140,6 +140,14 @@ typedef struct {
     bool allow_misalign;
 } riscv_io_t;
 
+#ifdef FUZZER
+// For fuzzing only: initialize the RISC-V emulator, and execute the given
+// binary stream
+int rv_init_and_execute_binary(const uint8_t *data,
```

Comment:
There is no need to prepare fuzzer specific API.

Reply:
How can we open a binary stream with the current system set up though?

Reply:
> * Take the entire logic out of the main function
> * Send in the fuzzer input by using the `argv`

It sounds better. Again, `riscv.h` is considered as a public header.